e2e/tests/transfer/forwarding_test.go

```diff
@@ -0,0 +1,94 @@
+package transfer
+
+import (
+	"context"
+	"testing"
+
+	testifysuite "github.com/stretchr/testify/suite"
+
+	"github.com/cosmos/ibc-go/e2e/testsuite"
+	"github.com/cosmos/ibc-go/e2e/testsuite/query"
+	"github.com/cosmos/ibc-go/e2e/testvalues"
+)
+
+func TestTransferForwardingTestSuite(t *testing.T) {
+	testifysuite.Run(t, new(TransferForwardingTestSuite))
+}
+
+type TransferForwardingTestSuite struct {
+	testsuite.E2ETestSuite
+}
+
+// TODO: replace this with actual tests
+// this test verifies that three chains can be set up, and the relayer will relay
+// packets between them as configured in the newInterchain function.
+func (s *TransferForwardingTestSuite) TestThreeChainSetup() {
+	ctx := context.TODO()
+	t := s.T()
+
+	relayer, channelA := s.SetupChainsRelayerAndChannel(ctx, s.TransferChannelOptions(), func(options *testsuite.ChainOptions) {
+		// copy all values of existing chains and tweak to make unique to new chain.
+		chainCSpec := *options.ChainSpecs[0] // nolint
```

Comment:
it was complaining about copying a lock, I thin we can ignore since we'll be refactoring this entire test later.

Reply:
Yeah that's a nice idea, I think we can do something similar to where we have `s.TransferChannelOptions()`. The 3rd chain will likely be the same in every test that needs it, so we can default to 2. Then provide a standard default for the 3 chain setup.